private/pkg/git/repository_test.go

```diff
@@ -66,19 +66,100 @@ func TestCommits(t *testing.T) {
 
 	repo := gittest.ScaffoldGitRepository(t)
 	var commits []git.Commit
-	err := repo.ForEachCommit(gittest.DefaultBranch, func(c git.Commit) error {
+	err := repo.ForEachCommit(func(c git.Commit) error {
 		commits = append(commits, c)
 		return nil
 	})
-
 	require.NoError(t, err)
 	require.Len(t, commits, 3)
+
 	assert.Equal(t, commits[0].Message(), "third commit")
 	assert.Contains(t, commits[0].Parents(), commits[1].Hash())
 	assert.Equal(t, commits[1].Message(), "second commit")
 	assert.Contains(t, commits[1].Parents(), commits[2].Hash())
 	assert.Equal(t, commits[2].Message(), "initial commit")
 	assert.Empty(t, commits[2].Parents())
+
+	t.Run("default_behavior", func(t *testing.T) {
+		var commitsFromDefaultBranch []git.Commit
```

Comment:
why are these not parallel?

Reply:
See comment in `.golancgi.yaml` [here](https://github.com/bufbuild/buf/pull/2371/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R216), it's because the git repo has a single object reader, not safe to use concurrently. I'll file a ticket to do that improvement.

Reply:
got it